service/controller/v26/resource/configmap/resource.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/randomkeys"
-	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
It is really cool that you do all this but when yo notice that there is more to it than one file you could think of providing a separate PR to make things easier.